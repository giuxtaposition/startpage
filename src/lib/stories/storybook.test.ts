import initStoryshots, {
    multiSnapshotWithOptions,
    Stories2SnapsConverter,
} from '@storybook/addon-storyshots'

initStoryshots({
    framework: 'svelte',
    stories2snapsConverter: new Stories2SnapsConverter({
        storiesExtensions: ['.svelte'],
        snapshotsDirName: '../../../../../__snapshots__',
    }),
    test: multiSnapshotWithOptions({}),
})
