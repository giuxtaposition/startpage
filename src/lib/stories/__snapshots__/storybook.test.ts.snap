// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Section Section 1`] = `
<section
  class="storybook-snapshot-container"
>
   
  <section
    class="links-section blue svelte-qtp5rb"
  >
    <h1
      class="svelte-qtp5rb"
    >
      section title
    </h1>
     
    <ul
      class="links svelte-qtp5rb"
    >
      <li
        class="link svelte-qtp5rb"
      >
        <a
          class="svelte-qtp5rb"
          href="https://github.com/giuxtaposition/startpage"
        >
          Link 1
        </a>
         
      </li>
      <li
        class="link svelte-qtp5rb"
      >
        <a
          class="svelte-qtp5rb"
          href="https://github.com/giuxtaposition/startpage"
        >
          Link 2
        </a>
         
      </li>
      <li
        class="link svelte-qtp5rb"
      >
        <a
          class="svelte-qtp5rb"
          href="https://github.com/giuxtaposition/startpage"
        >
          Link 3
        </a>
         
      </li>
    </ul>
  </section>
  
   
  
  
</section>
`;
